sdks/python/apache_beam/typehints/decorators.py

```diff
@@ -105,6 +107,7 @@ def foo((a, b)):
     'TypeCheckError',
 ]
 
+logger = logging.getLogger(__name__)
```

Comment:
It uses the module name instead of "root". See https://issues.apache.org/jira/browse/BEAM-3523 for details. Rereading that however, I realize that the worker expects us to log to root so I'll revert this for now.